package-lock.json

```diff
@@ -2324,9 +2325,9 @@
       "dev": true
     },
     "node_modules/typescript": {
-      "version": "3.8.0-dev.20200207",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-3.8.0-dev.20200207.tgz",
-      "integrity": "sha512-JICbjjqDfx99wFJSIrZLKWNsp2tmAi5NYazcNndcTjpwS1sFOtPwX94oJiEAFan75l0MFaBOWnxUMvyAM9M7ew==",
+      "version": "4.7.3",
```

Comment:
Hmm, maybe I shouldn't mess with the typescript version in package-lock?

Reply:
I think that's ok since the CI still passes